src/loader.js

```diff
@@ -1,16 +1,17 @@
 const callbackName = 'recaptchaOnLoad';
 const RECAPTCHA_URL = `https://www.google.com/recaptcha/api.js?onload=${callbackName}&render=explicit`;
+const SECONDARY_URL = `https:/www.recaptcha.net/recaptcha/api.js?onload=${callbackName}&render=explicit`;
```

Comment:
I think we are missing a slash here ```https:/www```

Reply:
good catch! chrome auto serolved the endpoint so i didnt see it. will push a minor fix